src/sliceDataStorage.h

```diff
@@ -102,6 +105,14 @@ class SliceLayer
     Polygons openPolyLines; //!< A list of lines which were never hooked up into a 2D polygon. (Currently unused in normal operation)
 
     /*!
+     * \brief The parts of the model that are exposed at the very top of the
+     * model.
+     *
+     * This is filled only when the top surface is needed.
+     */
+    TopSurface* top_surface = nullptr;
```

Comment:
I thought this kind of syntax caused problems for Windows...
I thought it had to be `TopSurface* top_surface(nullptr);`

Not sure, though...